output/formatter.go

```diff
@@ -112,6 +118,25 @@ func reportCSV(w io.Writer, data *reportInfo) error {
 	return nil
 }
 
+func reportJUnitXML(w io.Writer, data *reportInfo) error {
+	groupedData := groupDataByRules(data)
+	junitXMLStruct := createJUnitXMLStruct(groupedData)
+
+	raw, err := xml.MarshalIndent(junitXMLStruct, "", "\t")
+	if err != nil {
+		panic(err)
+	}
+
+	xmlHeader := []byte("<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n")
+	raw = append(xmlHeader, raw...)
+	_, err = w.Write(raw)
+	if err != nil {
+		panic(err)
```

Comment:
could we just return the error here instead of panic?

Reply:
opps. yes, should return an error here.